README.md

```diff
@@ -1 +1,966 @@
+  
+```
+   $ ballerina build 
+   
+   @kubernetes:Service                      - complete 1/1
+   @kubernetes:Ingress                      - complete 1/1
+   @kubernetes:Docker                       - complete 3/3 
+   @kubernetes:Deployment                   - complete 1/1
+  
+   Run following command to deploy kubernetes artifacts:  
+   kubectl apply -f ./target/phone_store_service/kubernetes
+   
+   @kubernetes:Service                      - complete 1/1
+   @kubernetes:Ingress                      - complete 1/1
+   @kubernetes:Docker                       - complete 3/3 
+   @kubernetes:Deployment                   - complete 1/1
+  
+   Run following command to deploy kubernetes artifacts:  
+   kubectl apply -f ./target/phone_order_delivery_service/kubernetes
+  
+   
+```
+
+- You can verify that the Docker image that we specified in `` @kubernetes:Deployment `` is created, by using `` docker images ``. 
+- Also the Kubernetes artifacts related our service, will be generated under `` ./target/phone_store_service/kubernetes`` and ``/target/phone_order_delivery_service/kubernetes``. 
+- Now you can create the Kubernetes deployment using:
+
+```bash
+   $ kubectl apply -f ./target/phone_store_service/kubernetes
+   
+   deployment.extensions "ballerina-guides-phone_store_service" created
+   ingress.extensions "ballerina-guides-phone_store_service" created
+   service "ballerina-guides-phone_store_service" created
+   
+   kubectl apply -f ./target/phone_order_delivery_service/kubernetes
+   
+   deployment.extensions "ballerina-guides-phone_order_delivery_service" created
+   ingress.extensions "ballerina-guides-phone_order_delivery_service" created
+   service "ballerina-guides-phone_order_delivery_service" created
+```
+
+- You can verify Kubernetes deployment, service and ingress are running properly, by using following Kubernetes commands. 
+
+```bash
+   $ kubectl get service
+   $ kubectl get deploy
+   $ kubectl get pods
+   $ kubectl get ingress
+```
+
+- If everything is successfully deployed, you can invoke the service either via Node port or ingress. 
+
+Node Port:
+```bash
+    curl -v -X POST -d \
+   '{"Name":"John", "Address":"20, Palm Grove, Colombo, Sri Lanka", 
+   "ContactNumber":"+94718930874", "PhoneName":"Apple:190000"}' \
+   "http://localhost:9090/phonestore/placeOrder" -H "Content-Type:application/json"  
+```
+
+Ingress:
+
+Add `/etc/hosts` entry to match hostname. 
+``` 
+   127.0.0.1 ballerina.guides.io
+```
+
+Access the service 
+```bash
+    curl -v -X POST -d \
+   '{"Name":"John", "Address":"20, Palm Grove, Colombo, Sri Lanka", 
+   "ContactNumber":"+94718930874", "PhoneName":"Apple:190000"}' \
+   "http://localhost:9090/phonestore/placeOrder" -H "Content-Type:application/json" 
+```
+## Observability 
```

Comment:
Please refer to https://github.com/ballerina-guides/restful-service/blob/master/README.md#observability and refine this section accordingly.

Reply:
done